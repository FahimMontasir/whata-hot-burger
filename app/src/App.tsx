import 'react-native-gesture-handler';
import React from 'react';
import {SafeAreaProvider} from 'react-native-safe-area-context';
import {NavigationContainer} from '@react-navigation/native';
import {View, Text} from 'react-native';
import styled from 'styled-components/native';
import {Button, Icon as I} from '@rneui/themed';
import Icon from 'react-native-vector-icons/FontAwesome';

interface IProps {
  name?: string;
}

const Yo = styled.Text<IProps>`
  color: red;
`;

const App = () => {
  return (
    <SafeAreaProvider>
      <NavigationContainer>
        <View>
          <Text>hello world</Text>
          <Yo>you</Yo>
          <Button title="hello" />

          <I
            raised
            name="home"
            type="font-awesome"
            color="#f50"
            onPress={() => console.log('hello')}
          />
          <Icon name="rocket" size={30} color="#4F8EF7" />
        </View>
      </NavigationContainer>
    </SafeAreaProvider>
  );
};
export default App;
