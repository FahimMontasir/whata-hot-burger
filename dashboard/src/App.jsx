import React from "react";
import { HelmetProvider } from "react-helmet-async";
import { SettingsProvider } from "./store/contexts/SettingsContext";
import { BrowserRouter } from "react-router-dom";
import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
import { ToastContainer } from "react-toastify";
import { Provider } from "react-redux";
import { CollapseDrawerProvider } from "./store/contexts/CollapseDrawerContext";
import { store } from "./store/redux/store";
import ThemeConfig from "./theme";
import ThemePrimaryColor from "./common/ThemePrimaryColor";

//routes
import Router from "./routes";

function App() {
  return (
    <HelmetProvider>
      <SettingsProvider>
        <CollapseDrawerProvider>
          <BrowserRouter>
            <Provider store={store}>
              <ThemeConfig>
                <ThemePrimaryColor>
                  <LocalizationProvider dateAdapter={AdapterDateFns}>
                    <Router />
                  </LocalizationProvider>
                </ThemePrimaryColor>
              </ThemeConfig>
              <ToastContainer />
            </Provider>
          </BrowserRouter>
        </CollapseDrawerProvider>
      </SettingsProvider>
    </HelmetProvider>
  );
}

export default App;
