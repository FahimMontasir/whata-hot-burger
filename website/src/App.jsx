import React from "react";
import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
import ThemeConfig from "./theme";
import Settings from "./common/settings";
import ThemePrimaryColor from "./common/ThemePrimaryColor";
import Router from "./routes";

function App() {
  return (
    <ThemeConfig>
      <ThemePrimaryColor>
        <Settings />
        <LocalizationProvider dateAdapter={AdapterDateFns}>
          <Router />
        </LocalizationProvider>
      </ThemePrimaryColor>
    </ThemeConfig>
  );
}

export default App;
