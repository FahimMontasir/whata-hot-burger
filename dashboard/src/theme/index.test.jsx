import ThemeConfig from "./index";
import renderer from "react-test-renderer";
import { render, screen } from "@testing-library/react";
import App from "../App";

describe("Theme", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<ThemeConfig />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("should debug console", () => {
    render(<App />);
    screen.debug();
  });
});
