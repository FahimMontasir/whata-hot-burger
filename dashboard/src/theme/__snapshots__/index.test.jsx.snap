// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theme should match snapshot 1`] = `null`;
